modules/add-contextual-data/add-contextual-data-grammar.ym

```diff
@@ -82,7 +82,7 @@ parser_add_contextual_data_opts
         ;
 
 parser_add_contextual_data_opt
-        : KW_DATABASE '(' string ')'
+        : KW_DATABASE '(' path_config ')'
```

Comment:
What is the difference between `path_db` and `path_config`? In other cases, you used `path_db`.

Reply:
Yes, it should be ```path_db```. There is no such difference between ```path_db``` and ```path_config```.